src/picongpu/submit/hypnos-hzdr/k80_profileRestart.tpl

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
+# Copyright 2013-2016 Axel Huebl, Anton Helm, Rene Widera, Richard Pausch
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+
+## calculation are done by tbg ##
+TBG_queue="k80"
+
+# settings that can be controlled by environment variables before submit
+TBG_mailSettings=${MY_MAILNOTIFY:-"n"}
+TBG_mailAddress=${MY_MAIL:-"someone@example.com"}
+TBG_author=${MY_NAME:+--author \"${MY_NAME}\"}
+
+# 8 gpus per node if we need more than 8 gpus else same count as TBG_tasks
+TBG_gpusPerNode=`if [ $TBG_tasks -gt 8 ] ; then echo 8; else echo $TBG_tasks; fi`
+
+#number of cores per parallel node / default is 2 cores per gpu on k20 queue
```

Comment:
K80 nodes: 4 K80 = 8 GPUs
32 "processor" IDs (Intel(R) Xeon(R) CPU E5-2630 v3): 2 sockets each 8 codes with 2x HT

since HT is enabled it should actually be 32/2/8 = `2 cores (4 HT threads) per GPU`
(`ppn` is configured to number of cores in this queue, having 16 per node available)